### ISSUE TITLE
Organize project member list

### ISSUE THREAD

**Issue by myieye** [sillsdev/languageforge-lexbox#379]
Currently there is no sort applied to project members and managers blend in with the others.

We probably want something like:
- Managers first
- Sort alphabetically
- Make managers visually distinct

**Comment by hahn-kev** [comment]
Decided to play with this because I thought I had an idea of how we could do it. I had to replace the badge that we use with a button to get it to respond to the size in css grid so that's why it looks weird.

But I couldn't do exactly what I wanted with CSS grid. Here's what I came up with:
![Image](https://github.com/sillsdev/languageforge-lexbox/assets/4575355/e0d64753-e46a-4595-a46b-3071311e1747)

This has a fixed number of columns, but the size of each column is dynamic based on the widest child. The last column is more narrow because it doesn't have any long names in it. However if there's not enough room for all 3 columns it'll just overflow, so that's not great.

Alternatively we if we can set a min size for a badge (200px in my example) we can do this where the number of columns is responsive to the screen size.
![Image](https://github.com/sillsdev/languageforge-lexbox/assets/4575355/2ee5a04a-f076-45b4-824b-eafdbbe278f2)
We just need to handle truncating long user names.

Finally, we could switch to this grid mode only when there's over 5 (tbd) users as it seems to work fine for a small number of users as is. This would just require a single css class to toggle the behavior.
